news.d/bugfix/1304.ui.md

```diff
@@ -0,0 +1 @@
+-Fix circular imports involving the translation macro and GUI forms.
```

Comment:
It's not a user visible change, and the description is wrong, so I suggest you just remove the fragment.

Reply:
Right, this PR's focus is changed; still an improvement but much more minor.

(and since it's not a big deal any more I get to see what a force push does to the arrangement of comments on a PR)